Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -368,27 +387,27 @@ extension PlaceCarouselViewController: EventsProviderDelegate {
 
 extension PlaceCarouselViewController: PlacesProviderDelegate {
     func placesProviderWillStartFetchingPlaces(_ controller: PlacesProvider) {
-        // TODO placeholder for the waiting state.
-        if self.places.count == 0 {
-            headerView.numberOfPlacesLabel.text = "Waiting"
-        }
     }
 
     func placesProviderDidFinishFetchingPlaces(_ controller: PlacesProvider) {
-        // no op
         if self.places.isEmpty {
             headerView.numberOfPlacesLabel.text = "No Places Found"
```

Comment:
I guess it depends if we want to use the loading error state as the empty state. I can see this being shown if we haven't received any places - like on first load - but might be strange if we are replacing cached sites with this.